src/walletConnect/screens/RequestContent.tsx

```diff
@@ -118,7 +115,7 @@ function RequestContent({
         )}
       </ScrollView>
 
-      <View style={styles.buttonContainer} pointerEvents={isLoading ? 'none' : undefined}>
+      <View style={styles.buttonContainer} pointerEvents={isLoading.current ? 'none' : undefined}>
```

Comment:
This could lead to issues. We should use the state instead.

Reply:
good point, i've updated